File: WeatherApp/app/src/main/java/com/example/android/Task03/Retrofit/ApiInterface.java
package com.example.android.Task03.Retrofit;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;


public interface ApiInterface {
    @GET("forecast?appid=20643d0926a7e824ed846f8a8ab41274&units=metric")
    Call<ListData> getWeatherData(@Query("q") String name);
}

File: WeatherApp Kotlin/app/src/main/java/com/example/android/weatherApp/retrofit/ApiInterface.kt
package com.example.android.weatherApp.retrofit

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiInterface {
    @GET("forecast?appid=20643d0926a7e824ed846f8a8ab41274&units=metric")
    fun getWeatherData(@Query("q") name: String?): Call<ListData?>?
}